internal/compiler/typeregister.rs

```diff
@@ -84,13 +86,26 @@ pub(crate) const RESERVED_DROP_SHADOW_PROPERTIES: &[(&str, Type)] = &[
     ("drop-shadow-color", Type::Color),
 ];
 
+pub(crate) const RESERVED_ACCESSIBILITY_PROPERTIES: &[(&str, Type)] = &[
+    //("accessible-role", ...)
+    ("accessible-checked", Type::Bool),
+    ("accessible-delegate-focus", Type::String),
```

Comment:
```suggestion
    ("accessible-delegate-focus", Type::Int32),
```

That property needs to have some internal comment about what this is.
Also if we keep it internal, we should make sure that it doesn't appear in the auto-completion. Actually, it should be an error to use it. Like `Clip` and stuff.

Reply:
Can this really be a hard error, considering that I have to use it in the tab bar?

Reply:
You can check the span if it is in a builtin.  We do that with other internal stuff like all the Native* items.

Reply:
> Int32 auto-converts to String in the slint language, once I change this to Int32, we have to do the same on the rust side;-)

The compiler already has code to convert Int32 and Float32 to string so that you don't have to do anything on the rust side.

Reply:
The compiler has that code and uses that while we have `Type::String` above.

It stops doing so once we change that and then we need to handle that similar to how we already handle the bools when turning the property into a string behind the type systems back. But yes, the code is there, it just needs to be wired up correctly.

Reply:
Cool,changing the type works now. It failed when I tried that last.